TFT/src/User/API/Language/language_gr.h

```diff
@@ -342,5 +342,6 @@
     #define STRING_EXT_TEMPLOW            "Hotend temperature is below\nminimum temperature (%d℃)."
     #define STRING_HEAT_HOTEND            "Heat hotend to %d℃?"
     #define STRING_Z_ALIGN                "Z Align"
+    #define STRING_TEST_LEVELING          "test Leveling"
```

Comment:
Use proper sentence case for adding new strings
```suggestion
    #define STRING_TEST_LEVELING          "Test leveling"
```